hemera_udf/aave_v2/export_aave_v2_job.py

```diff
@@ -0,0 +1,408 @@
+                        parameters=[a_record.aave_user],
+                        block_number=a_record.block_number,
+                    )
+                )
+            elif a_record.type() == AaveV2WithdrawD.type():
+                reserve = self.reserve_dic[a_record.reserve]
+                eth_call_lis.append(
+                    Call(
+                        target=reserve.a_token_address,
+                        function_abi=SCALED_BALANCE_OF_FUNCTION,
+                        parameters=[a_record.aave_user],
+                        block_number=a_record.block_number,
+                    )
+                )
+            elif a_record.type() == AaveV2RepayD.type() or a_record.type() == AaveV2BorrowD.type():
+                reserve = self.reserve_dic[a_record.reserve]
+                eth_call_lis.append(
+                    Call(
+                        target=reserve.stable_debt_token_address,
+                        function_abi=PRINCIPAL_BALANCE_OF_FUNCTION,
+                        parameters=[a_record.aave_user],
+                        block_number=a_record.block_number,
+                    )
+                )
+                eth_call_lis.append(
+                    Call(
+                        target=reserve.variable_debt_token_address,
+                        function_abi=SCALED_BALANCE_OF_FUNCTION,
+                        parameters=[a_record.aave_user],
+                        block_number=a_record.block_number,
+                    )
+                )
+
+            elif a_record.type() == AaveV2LiquidationCallD.type():
+                aave_user = a_record.aave_user
+                collateral_asset = a_record.collateral_asset
+                collateral_reserve = self.reserve_dic[collateral_asset]
+                eth_call_lis.append(
+                    Call(
+                        target=collateral_reserve.a_token_address,
+                        function_abi=SCALED_BALANCE_OF_FUNCTION,
+                        parameters=[aave_user],
+                        block_number=a_record.block_number,
+                    )
+                )
+                debt_asset = a_record.debt_asset
+                debt_reserve = self.reserve_dic[debt_asset]
+
+                eth_call_lis.append(
+                    Call(
+                        target=debt_reserve.stable_debt_token_address,
+                        function_abi=PRINCIPAL_BALANCE_OF_FUNCTION,
+                        parameters=[aave_user],
+                        block_number=a_record.block_number,
+                    )
+                )
+                eth_call_lis.append(
+                    Call(
+                        target=debt_reserve.variable_debt_token_address,
+                        function_abi=SCALED_BALANCE_OF_FUNCTION,
+                        parameters=[aave_user],
+                        block_number=a_record.block_number,
+                    )
+                )
```

Comment:
Yes, in fact, for every address at one reserve (underlying token), there is only one borrow mode at a time: 1 for variable and 2 for stable. Here, I add both into rpc_calls in case any unexpected data appears. Additionally, I think the cost is acceptable.